test/e2e/storage/flexvolume.go

```diff
@@ -197,11 +196,6 @@ var _ = utils.SIGDescribe("Flexvolumes", func() {
 
 		testFlexVolume(ctx, driverInstallAs, config, f)
 
-		ginkgo.By("waiting for flex client pod to terminate")
-		if err := e2epod.WaitForPodTerminatedInNamespace(ctx, f.ClientSet, config.Prefix+"-client", "", f.Namespace.Name); !apierrors.IsNotFound(err) {
-			framework.ExpectNoError(err, "Failed to wait client pod terminated: %v", err)
```

Comment:
The commit message for this change explains it:

https://github.com/kubernetes/kubernetes/pull/113298/commits/b3366ce8952daae4a69865134e3180825107cebc

> Calling WaitForPodTerminatedInNamespace after testFlexVolume is useless because
> the client pod that it waits for always gets deleted by testVolumeClient:
> 
> https://github.com/kubernetes/kubernetes/blob/0fcc3dbd55732c501d4948840f639e00a2fe33c5/test/e2e/framework/volume/fixtures.go#L541-L546
> 
> Worse, because WaitForPodTerminatedInNamespace treats "not found" as "must keep
> polling", these two tests always kept waiting for 5 minutes:
> 
>     Kubernetes e2e suite: [It] [sig-storage] Flexvolumes should be mountable
>     when non-attachable 	6m4s
> 
> The only reason why these tests passed is that WaitForPodTerminatedInNamespace
> used to return the "not found" API error. That is not guaranteed and about to
> change.